src/routes/user.route.ts

```diff
@@ -269,8 +269,9 @@ usersRouter.patch('/users/changepassword', userAuthJWT, UserController.changePas
 usersRouter.patch('/users/update', userAuthJWT, UserController.updateUser);
 usersRouter.delete('/users/delete/:id', UserController.deleteUser);
 usersRouter.post('/login', loginByGoogle);
+usersRouter.post('/register/google', registerByGoogle)
 usersRouter.post('/forgot', UserController.forgotPassword);
-usersRouter.get('/reset', UserController.resetPassword);
+usersRouter.post('/reset', UserController.resetPassword);
```

Comment:
I can see you changed the request to post which was a good idea as we were sending a password change